app.js

```diff
@@ -44,9 +40,10 @@ app.get(['/favicon.ico', '/robots.txt'], (req, res) => {
 
 app.get('/search', (req, res) => {
   loadDatabase((allItems) => {
+    console.log(allItems);
     res.render('search', {
-      matches: searchDatabase(req.query, allItems)
-        .sort((a, b) => (a.floor === b.floor ? 0 : +(a.floor > b.floor) || -1)),
+      matches: searchDatabase(req.query, allItems).sort((a, b) =>
+        (a.floor === b.floor ? 0 : +(a.floor > b.floor) || -1)),
     });
```

Comment:
this is noise, refrain from doing cosmetic changes.

Reply:
you may understand why if you try the git blame command